vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -118,7 +122,7 @@ def fit(self,
                 # Recommended for applications that benefit from the best possible model accuracy.
                 best_quality={
                     'hyperparameters': {
-                        'model': Categorical('resnet50_v1b', 'resnet101_v1d', 'resnest200'),
+                        'model': Categorical('coat_lite_small', 'twins_pcpvt_base', 'swin_base_patch4_window7_224'),
                         'lr': Real(1e-5, 1e-2, log=True),
                         'batch_size': Categorical(8, 16, 32, 64, 128),
```

Comment:
Please double check if batch_size 128 causes the full system to crash (and not continue doing trials). I suspect it does.